scripts/monorepo/with-progress.js

```diff
@@ -0,0 +1,17 @@
+const ora = require('ora');
+
+async function withProgress(text, fn, ...args) {
+  spinner = ora(text).start();
```

Comment:
TIL about `ora` :)

Reply:
If I have to run something multiple times, I just had to have some pretty spinners to stare at at the very least 😆